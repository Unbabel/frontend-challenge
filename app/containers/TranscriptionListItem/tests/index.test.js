/**
 * Test the repo list item
 */

import React from 'react';
import { render } from '@testing-library/react';
import { IntlProvider } from 'react-intl';

import { TranscriptionListItem } from '../index';

const renderComponent = (props = {}) =>
  render(
    <IntlProvider locale="en">
      <TranscriptionListItem {...props} />
    </IntlProvider>,
  );

describe('<TranscriptionListItem />', () => {
  let item;

  // Before each test reset the item data for safety
  beforeEach(() => {
    item = {
      id: 1,
      voice: 'Voice 2',
      text: 'This is a one line sentence made to show how it fits',
    };
  });

  it('should render a ListItem', () => {
    const { container } = renderComponent({ item });
    expect(container.firstChild).toMatchSnapshot();
  });
});
