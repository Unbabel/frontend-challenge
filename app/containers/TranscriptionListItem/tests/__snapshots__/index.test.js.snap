// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TranscriptionListItem /> should render a ListItem 1`] = `
<li
  class="Wrapper-sc-14y9d5o-0 iLHUmP"
>
  <div
    class="Item-sc-1wdy6vf-0 cuMxiu"
  >
    <div
      class="Wrapper-nauwn6-0 kawEeG"
    />
  </div>
</li>
`;
